test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Culture.Tests.ps1

```diff
@@ -1,25 +1,64 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-Describe "Get-Culture DRT Unit Tests" -Tags "CI" {
-    It "Should works proper with get-culture" {
-        $results = get-Culture
-        $results -is "System.Globalization.CultureInfo" | Should -BeTrue
-        $results[0].Name | Should -Be $PSCulture
-    }
-}
 
 Describe "Get-Culture" -Tags "CI" {
 
     It "Should return a type of CultureInfo for Get-Culture cmdlet" {
 
-	Get-Culture | Should -BeOfType CultureInfo
+        Get-Culture | Should -BeOfType [CultureInfo]
 
     }
 
     It "Should have $ culture variable be equivalent to (Get-Culture).Name" {
 
-	(Get-Culture).Name | Should -Be $PsCulture
+        (Get-Culture).Name | Should -Be $PsCulture
+
+    }
+
+    It "Should return the specified culture with '-Name' parameter" {
+
+        $ci = Get-Culture -Name ru-RU
+        $ci | Should -BeOfType [CultureInfo]
+        $ci.Name | Should -BeExactly "ru-RU"
+    }
+
+    It "Should return specified cultures with '-Name' parameter" {
 
+        $ciArray = Get-Culture "", "ru-RU"
+        $ciArray | Should -BeOfType [CultureInfo]
+        ,$ciArray | Should -BeOfType [System.Array]
+        $ciArray | Should -HaveCount 2
+        $ciArray[0] | Should -BeOfType [CultureInfo]
+        $ciArray[0].LCID | Should -Be 127
+        $ciArray[1] | Should -BeOfType [CultureInfo]
+        $ciArray[1].LCID | Should -Be 1049
     }
 
+    It "Should accept values from a pipeline for '-Name' parameter" {
+
+        $ciArray = "", "ru-RU" | Get-Culture
+        $ciArray | Should -BeOfType [CultureInfo]
+        ,$ciArray | Should -BeOfType [System.Array]
+        $ciArray | Should -HaveCount 2
+        $ciArray[0] | Should -BeOfType [CultureInfo]
+        $ciArray[0].LCID | Should -Be 127
+        $ciArray[1] | Should -BeOfType [CultureInfo]
+        $ciArray[1].LCID | Should -Be 1049
+    }
+
+    It "Should return the culture array with '-ListAvailable' parameter" {
+
+        $ciArray = Get-Culture -ListAvailable
+        ,$ciArray | Should -BeOfType [System.Array]
+        $ciArray.Count | Should -BeGreaterThan 0
+        $ciArray[0] | Should -BeOfType [CultureInfo]
+    }
+
+    It "Should write an error on unsupported culture name" -Skip:(!$isWindows) {
```

Comment:
I suggest adding the issue link as a comment for -Skip

https://github.com/dotnet/corefx/issues/6374

Reply:
I updated the test - now it throw on Unix-es too. Link to useful comment added too.